### PR TITLE
Feat/boilerplate

### PR DESCRIPTION
**Pull request by aricalves**
## Boilerplate, sweet boilerplate.

This PR is packed with all the good stuff:
- Babel setup
- Environmental Variable config
- Eslint things
- `build`, `start`, and `crawl` scripts
- Readme updates
- Installs a few dependencies to be used later

[Trello Ticket](https://trello.com/c/yDHUcdzH)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import app from './app';
+
+require('dotenv').config();
```
Comment (andino93) [inline src/index.js]: If you want to use an import statement here, you can do `import {} from 'dotenv/config'` and it'll achieve the same thing without having use a require statement

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+const { PORT = 3000 } = process.env;
+
+app.listen(PORT, () => console.log(`( ͡° ͜ʖ ͡°)☞  ${PORT}`)); // eslint-disable-line
```
Comment (andino93) [inline src/index.js]: remove console log for prod code

File: src/index.js (hunk 1 above)
Comment (aricalves) [reply src/index.js]: Nice! Will fix.

File: src/index.js (hunk 2 above)
Comment (aricalves) [reply src/index.js]: Whoops, thanks

**Review by andino93 — APPROVED** [review]

**Pull request merged**